gem/lib/pupperware/spec_helper.rb

```diff
@@ -205,14 +205,15 @@ def docker_compose_preload_cert_volumes()
       STDOUT.puts("Pre-loading certificates for service #{service_name}")
       docker_volume_cp(src_path: source, dest_volume: volume, dest_dir: dest_dir,
         uid: labels ? labels['com.puppet.certs.uid'] : nil,
-        gid: labels ? labels['com.puppet.certs.gid'] : nil)
+        gid: labels ? labels['com.puppet.certs.gid'] : nil,
+        dir_mode: '700', file_mode: '600')
```

Comment:
I want to do a little more testing, but I believe this works @jpartlow

Reply:
I also have another variation that only touches the mode when you've specified a uid, which is a bit more magical... but think maybe I should just do something like this:

```
dir_mode: labels ? labels['com.puppet.certs.dir_mode'] : '700',
file_mode: labels ? labels['com.puppet.certs.file_mode'] : '600'
```

Thoughts @jpartlow ?

Reply:
Yes, if there is no label, then no mode should be set.

Reply:
Yeah, just for the tests... trying to keep it as simple / robust as possible I guess.